tests/typing/api.py

```diff
@@ -94,6 +96,14 @@ def exc_tune_waiting_timedelta() -> None: ...
 
 set_active(False)
 
+set_testing(True)
+set_testing(True, attempts=2)
+
+if is_testing() is False:
```

Comment:
Just curious, is there a reason why you're testing via `is False` here rather than `not is_testing()`?